src/Formats/FormatSettings.h

```diff
@@ -46,6 +46,8 @@ struct FormatSettings
     bool try_infer_dates = false;
     bool try_infer_datetimes = false;
 
+    bool output_format_markdown_escape_special_characters = false;
```

Comment:
It's better to place this setting to a struct with name `markdown` as it's done for other formats. Only general settings that are used for more than one format can be placed like this. I will change it